### DISCUSSION TITLE
1000 Warnings

### DISCUSSION THREAD

**Discussion by lublak** [valentjn/vscode-ltex#691] [Q&A]
Currently I am being slain by ltex with over 1000 errors.
![image](https://user-images.githubusercontent.com/44057030/186381675-c49e4646-8db5-4db6-8473-54337be99c5e.png)
![image](https://user-images.githubusercontent.com/44057030/186382319-2dd97448-3acd-4f3f-8161-1d2862b4150c.png)
In the process, everything comes with a warning, whether it is text or not. It even checks the included packages to see if they are spelled correctly:
![image](https://user-images.githubusercontent.com/44057030/186381923-c6fd061e-f7fc-4b01-a1c0-d21bf13f8e26.png)
![image](https://user-images.githubusercontent.com/44057030/186382118-85234fdf-3510-4e00-8c03-14d3ec55153b.png)
![image](https://user-images.githubusercontent.com/44057030/186382155-8dedaf58-0aa4-49a1-b82f-1d7e5bc3f08c.png)
![image](https://user-images.githubusercontent.com/44057030/186382192-2eb5d197-e4eb-4e5f-b8c6-d7f25d3ac5b4.png)

**Comment by Tsaaim** [comment]
I'm facing the same problem.
A workaround would be [this answer.](https://github.com/valentjn/vscode-ltex/discussions/838#discussioncomment-5825524)
You have to set _"ltex.diagnosticSeverity": "hint"_ in the settings.json.
It only hides the errors in the problems tab though.
